source/include-link.txt

```diff
@@ -0,0 +1,209 @@
+.. _cpp-include-link:
+
+===========================================
+Include and Link the Driver in Your Program
+===========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+Overview
+--------
+
+In this guide you will learn how to use the {+driver-short+} in your own projects. Examples are provided for using the {+driver-short+} with CMake and with ``pkg-config``.
+
+Example Code
+~~~~~~~~~~~~
+
+The examples on this page use the ``view_and_value.cpp`` example program from the {+driver-short+} source code. Obtain the {+driver-short+} source code by following the :ref:`cpp-quick-start-download-and-install` guide, if you have not previously done so.
+
+You do not need to run this program on a MongoDB server.
+
+Library References
+~~~~~~~~~~~~~~~~~~
+
+The examples use references to components of the {+driver-short+}. For CMake projects, the ``mongo::bsoncxx_shared`` target is used. For pkg-config projects, the ``libbsoncxx`` package is used. You may use the following library targets, depending on the needs of your project:
+
+- CMake
+  - ``mongo::bsoncxx_shared``
+  - ``mongo::mongoccxx_shared``
+  - ``mongo::bsoncxx_static``
+  - ``mongo::mongoccxx_static``
+- pkg-config
+  - ``libbsoncxx``
+  - ``libmongocxx``
+  - ``libbsoncxx-static``
+  - ``libmongocxx-static``
+
+The availability of targets is dependent on the particular installation method.
+
+In the sections that follow, replace ``<path-to-mongo-cxx-driver-sources>`` with the actual path where the {+driver-short+} source tree is located on your system.
+
+CMake
+-----
+
+You can use the {+driver-short+} through CMake. CMake provides the ``find_package`` command, which your project can use to locate the {+driver-short+} once it is installed. Alternatively, your project can use the advanced ``add_subdirectory`` command without requiring installation of the {+driver-short+}.
+
+With Driver Installation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Installing the {+driver-short+} enables using it in a simple and straighforward way. After installing, you can use CMake's ``find_package`` command to integrate the {+driver-short+} with your project. To learn how to install the {+driver-short+}, visit the following guides:
+
+- :ref:`cpp-quick-start-download-and-install` to install from source
+- :ref:`cpp-installation-advanced` to install from packages
+
+To use the ``find_package`` command, create a ``CMakeLists.txt`` file in your project directory. The following example creates a ``CMakeLists.txt`` file in the ``/home/user/project1`` project directory that uses ``find_package``:
+
+.. literalinclude:: /includes/cmake_with_driver_installation.txt
+  :caption: /home/user/project1/CMakeLists.txt
+  :start-after: -- sphinx-include-start --
+
+Then, you can build and run your project. The execution should produce output resembling the following sample output:
```

Comment:
S: Since this code example includes multiple commands and expected output, I'd break it up, and maybe omit some of the expected output. The structure could look something like:

Then, run the following commands to build your project:

code block:
cd /home/user/project1
cmake -Bbuild
cmake --build build

These commands return information about the build process. After the build completes, run the following command to run the project:

code block:
./build/view_and_value

The command line output resembles the following code:

code block:
the expected code output